trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts

```diff
@@ -70,11 +70,11 @@ describe(page, () => {
     wrapper.expectText(`${premiumPaymentsTabSelector} h2[data-cy="premium-payments-heading"]`).toRead('Premium payments');
     wrapper
       .expectText(`${premiumPaymentsTabSelector} p`)
-      .toMatch(/Enter received payments against reported fees by selecting them and then selecting the 'Add a payment' button./);
+      .toMatch(/Received payments are entered against reported fees through selection and then selection of the 'Add a payment' button./);
```

Comment:
Strange that we're not using page selectors or `cy.assertText` here 😞 

Appreciate this PR is purely content changes though.

Reply:
I would like to add e2e tests for this section later for the page content